backend/src/main/java/ca/bc/gov/hlth/hnweb/converter/hl7v3/GetDemographicsConverter.java

```diff
@@ -0,0 +1,122 @@
+package ca.bc.gov.hlth.hnweb.converter.hl7v3;
+
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ca.bc.gov.hlth.hnweb.model.GetPersonDetailsResponse;
+import ca.bc.gov.hlth.hnweb.model.StatusEnum;
+import ca.bc.gov.hlth.hnweb.model.v3.GetDemographicsRequest;
+import ca.bc.gov.hlth.hnweb.model.v3.GetDemographicsResponse;
+import ca.bc.gov.hlth.hnweb.model.v3.MessageMetaData;
+import ca.bc.gov.hlth.hnweb.model.v3.Name;
+import ca.bc.gov.hlth.hnweb.security.SecurityUtil;
+import ca.bc.gov.hlth.hnweb.security.UserInfo;
+import ca.bc.gov.hlth.hnweb.serialization.HL7Config;
+import ca.bc.gov.hlth.hnweb.serialization.HL7Serializer;
+import ca.bc.gov.hlth.hnweb.util.V2MessageUtil;
+import ca.bc.gov.hlth.hnweb.util.V3MessageUtil;
+
+/**
+ * Converter class for V3 messages Contains methods to facilitate converter a
+ * GetPersonDeatilsRequest to and GetDemographics and from
+ * GetDemographicsRsponse to a GetPersonDeatilsResponse
+ *
+ */
+
+public class GetDemographicsConverter {
+
+	private static final String WARNING = "Warning";
+	private static final Logger logger = LoggerFactory.getLogger(GetDemographicsConverter.class);
+	private static final String MRN_SOURCE = "MOH_CRS";
+
+	/**
+	 * @param phn
+	 * @return
+	 */
+	public GetDemographicsRequest convertRequest(String phn) {
+		logger.debug("Get Demographics details for PHN [{}]", phn);
+
+		GetDemographicsRequest demographicsRequest = buildDemographicsRequest(phn);
```

Comment:
I don't think there's any point in having separate buildDemographicsRequest method as it's only called by convertRequest and convertRequest doesn't do anything else. But there's nothing wrong with it either. Your choice.